cms/envs/common.py

```diff
@@ -264,7 +264,10 @@
 
     # Whether archived courses (courses with end dates in the past) should be
     # shown in Studio in a separate list.
-    'ENABLE_SEPARATE_ARCHIVED_COURSES': True
+    'ENABLE_SEPARATE_ARCHIVED_COURSES': True,
+
+    # Whether section-level highlights are enabled on the platform.
+    'ENABLE_SECTION_HIGHLIGHTS': False,
```

Comment:
I think this is fine, but I'm curious why a feature flag as opposed to waffle?

Reply:
Good point.  I'll change it.  Not sure why I defaulted to that.  Maybe cause I saw some other Studio code using the old way.

Reply:
Addressed in https://github.com/edx/edx-platform/pull/16316.